src/control/PlanarArmController.h

```diff
@@ -65,13 +66,20 @@ template <unsigned int N> class PlanarArmController {
 
 		// bounds check (new pos + vel vector <= sum of joint lengths)
 		double radius = kin.getSegLens().sum();
+		Eigen::Vector2d constrainedPos = pos;
```

Comment:
Don't think there's any point in having `constrainedPos`, let's delete this and just modify `pos` inside the if statement.